pinot-query-runtime/src/test/resources/queries/Distincts.json

```diff
@@ -0,0 +1,47 @@
+{
+  "general_distinct": {
+    "tables": {
+      "tbl": {
+        "schema": [
+          { "name": "intCol", "type": "INT" },
+          { "name": "doubleCol", "type": "DOUBLE" },
+          { "name": "strCol", "type": "STRING" },
+          { "name": "boolCol", "type": "BOOLEAN" }
+        ],
+        "inputs": [
+          [ 2, 300, "a", true ],
+          [ 2, 400, "a", true ],
+          [ 3, 100, "b", false ],
+          [ 100, 1, "b", false ],
+          [ 101, 1.01, "c", false ],
+          [ 150, 1.5, "c", false ],
+          [ 175, 1.75, "c", true ]
+        ]
+      }
+    },
+    "queries": [
+      { "sql":  "SELECT DISTINCT intCol FROM {tbl}" },
```

Comment:
Can we add 
select distinct intCol, intCol FROM {tbl1} 
select distinct col, 1, 2, 3 FROM {tbl1}